tsconfig.json

```diff
@@ -15,11 +15,10 @@
     "node_modules"
   ],
   "files": [
-    "./buidler.config.ts",
-    "./node_modules/@nomiclabs/buidler-ethers/src/type-extensions.d.ts",
-    "./node_modules/@nomiclabs/buidler-waffle/src/type-extensions.d.ts",
-    "./node_modules/buidler-typechain/src/type-extensions.d.ts",
-    "./node_modules/buidler-gas-reporter/src/type-extensions.d.ts",
-    "./node_modules/buidler-deploy/src/type-extensions.d.ts"
+    "./hardhat.config.ts",
+    // "./node_modules/@nomiclabs/buidler-ethers/src/type-extensions.d.ts",
```

Comment:
why is this commented out?

Reply:
just forgot to delete - I removed it now